scheduler/test/cook/test/rest/api.clj

```diff
@@ -470,6 +470,14 @@
                              :headers {"Content-Type" "application/json"}
                              :authorization/user "dgrnbrg"
                              :body-params {"jobs" [(job {"request" 2.5})] "pool" "test-pool"}})))))
+      (testing "Int request of disk valid"
+        (is (= 201
+               (:status (h {:request-method :post
+                            :scheme :http
+                            :uri "/jobs"
+                            :headers {"Content-Type" "application/json"}
+                            :authorization/user "dgrnbrg"
+                            :body-params {"jobs" [(job {"request" 20.0})] "pool" "test-pool"}})))))
```

Comment:
Shouldn't the request be an int for this test case?

Reply:
oops! a typo, just changed